components/brave_welcome_ui/brave_welcome.tsx

```diff
@@ -1,48 +1,38 @@
-/* This Source Code Form is subject to the terms of the Mozilla Public
- * License, v. 2.0. If a copy of the MPL was not distributed with this file,
- * You can obtain one at https://mozilla.org/MPL/2.0/. */
+// Copyright (c) 2022 The Brave Authors. All rights reserved.
+// This Source Code Form is subject to the terms of the Mozilla Public
+// License, v. 2.0. If a copy of the MPL was not distributed with this file,
+// you can obtain one at https://mozilla.org/MPL/2.0/.
 
 import * as React from 'react'
 import { render } from 'react-dom'
-import { bindActionCreators } from 'redux'
-import { Provider } from 'react-redux'
+import { initLocale } from 'brave-ui'
 
-import welcomeDarkTheme from './theme/welcome-dark'
-import welcomeLightTheme from './theme/welcome-light'
-import BraveCoreThemeProvider from '../common/BraveCoreThemeProvider'
+import { loadTimeData } from '$web-common/loadTimeData'
+import darkTheme from './theme/welcome-dark'
+import lightTheme from './theme/welcome-light'
```

Comment:
Is anything in the old custom themes (welcome-dark and welcome-light) actually used in this new version, or can we just stick to the default themes?